index.js

```diff
@@ -67,6 +81,31 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			var progressEnabled = progress.emitter.listeners('progress').length > 0;
+			if (progressEnabled) {
```

Comment:
@YurySolovyov Sorry, `if (size !== null) {` was based on a misconception, and doesn't make sense.